<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mySql</class>
 <widget class="QWidget" name="mySql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birthLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insetpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 25, 0); /* 文字颜色为白色 */
font: 700 10pt &quot;Microsoft Uighur&quot;;</string>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>631</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:rgb(0, 0, 0);
border-radius: 15px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>查询2</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>511</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(216, 67, 21, 0.7); /* 紫红色，透明度为0.7 */
    color: #fff; /* 文字颜色为白色 */
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    font: 28pt &quot;Mistral&quot;;
}


QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}

QLabel{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;

	font: 24pt &quot;Lucida Calligraphy&quot;;
    border-style: solid;
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 25, 0); /* 文字颜色为白色 */
font: 700 10pt &quot;Microsoft Uighur&quot;;</string>
   </property>
   <property name="text">
    <string>序号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(0, 25, 0); /* 文字颜色为白色 */
font: 700 10pt &quot;Microsoft Uighur&quot;;</string>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>1</y>
     <width>701</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/13.jpg);
QPushButton {
    background-color: rgba(216, 67, 21, 0.7); /* 紫红色，透明度为0.7 */
    color: #fff; /* 文字颜色为白色 */
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    font: 28pt &quot;Mistral&quot;;
}


QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}

QLabel{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;

	font: 24pt &quot;Lucida Calligraphy&quot;;
    border-style: solid;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: rgb(232, 180, 210);
    color: #fff;
    border-radius: 15px;
    border-style: dotted;
    border-color: rgb(170, 170, 255);
    border: 2px solid rgb(255, 255, 255);
    
	font: 14pt &quot;Segoe Script&quot;;
}

QPushButton:hover {
    background: #E57373; /* 设置鼠标悬停时的背景色 */
    border-color: rgb(255, 87, 34); /* 设置鼠标悬停时的边框颜色 */
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>findpushButton</zorder>
  <zorder>birthLineEdit</zorder>
  <zorder>insetpushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_4</zorder>
  <zorder>tableView</zorder>
  <zorder>pushButton</zorder>
  <zorder>imagelabel</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>idLineEdit</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
